common/bytes.go

```diff
@@ -59,3 +62,42 @@ func EnsureEnoughSize(in []byte, size int) []byte {
 	}
 	return in[:size] // Reuse the space if it has enough capacity
 }
+
+func FromHex(s string) []byte {
```

Comment:
These functions should be moved to `package hexutility`

Reply:
There is some hex package: https://github.com/ledgerwatch/erigon-lib/tree/main/common/hex